splinterd/Cargo.toml

```diff
@@ -132,6 +132,8 @@ database-sqlite = ["splinter/sqlite"]
 health-service = ["health"]
 https-bind = ["splinter/https-bind"]
 node = [
+    "authorization",
```

Comment:
This should probably pull in the libsplinter dependencies directly, since this local features are there mostly to guard splinterd code fragments.

Reply:
The daemon code won't compile, if it is just the splinter features, so you can't compile with just `--features node` without it